### PR TITLE
fix: two try its on one page

### PR DESCRIPTION
**Pull request by mpodlasin**
Quickfix, because we broke prod of platform 😨

### CODE REVIEW

**Review by mnaumanali94 — APPROVED** [review]

**Review by paulatulis — APPROVED** [review]

File: packages/elements-dev-portal/src/version.ts [hunk 1]
```diff
@@ -1,2 +1,2 @@
 // auto-updated during build
-export const appVersion = '1.5.1';
+export const appVersion = '1.6.2';
```
Comment (marbemac) [inline packages/elements-dev-portal/src/version.ts:2]: What's this?

Comment (Nezteb) [reply packages/elements-dev-portal/src/version.ts:2]: I've seen this get auto-bumped before too, but I have no idea what causes it. 🤔

File: packages/elements-core/src/components/Docs/Article/Article.stories.tsx [hunk 2]
```diff
@@ -2,6 +2,8 @@
 import basic from '../../../__fixtures__/articles/basic.md';
 // @ts-ignore
 import kitchenSink from '../../../__fixtures__/articles/kitchen-sink.md';
+// @ts-ignore
+import twoTryIts from '../../../__fixtures__/articles/two-try-its.md';
```
Comment (marbemac) [inline packages/elements-core/src/components/Docs/Article/Article.stories.tsx:6]: `multipleTryIts` :P

Comment (mpodlasin) [reply packages/elements-core/src/components/Docs/Article/Article.stories.tsx:6]: Oh come on 😂

**Comment by marbemac** [comment]
Yalc'ing into platform to verify

File: packages/elements-core/src/components/MarkdownViewer/CustomComponents/CodeComponent.tsx [hunk 3]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable prettier/prettier */
+import { PersistenceContextProvider } from '@stoplight/elements-core';
```
Comment (marbemac) [inline packages/elements-core/src/components/MarkdownViewer/CustomComponents/CodeComponent.tsx:2]: bad import

File: packages/elements-dev-portal/src/version.ts (hunk 1 above)
Comment (mpodlasin) [reply packages/elements-dev-portal/src/version.ts:2]: This is autogenerated during release process anyways - so what is in this file in src doesn't really matter.

Used to communicate the version of the library to `elements-dev-portal`.

File: packages/elements-core/src/components/MarkdownViewer/CustomComponents/CodeComponent.tsx [hunk 4]
```diff
@@ -73,7 +75,9 @@ export const CodeComponent: CustomComponentMapping['code'] = props => {
     }
 
     return (
-      <TryIt httpOperation={isHttpOperation(parsedValue) ? parsedValue : parseHttpRequest(parsedValue)} embeddedInMd />
+      <PersistenceContextProvider>
```
Comment (marbemac) [inline packages/elements-core/src/components/MarkdownViewer/CustomComponents/CodeComponent.tsx:78]: Pretty sure this should only be around the embedded try it though right?

Comment (marbemac) [reply packages/elements-core/src/components/MarkdownViewer/CustomComponents/CodeComponent.tsx:78]: ah I see the issue you prob ran into, the state is accessed up top in try it component, perhaps just an if/else here? can refactor this whole component in a later pr as we were discussing

Comment (mpodlasin) [reply packages/elements-core/src/components/MarkdownViewer/CustomComponents/CodeComponent.tsx:78]: What do you mean?

This is only used for embedding in markdown - see the `embeddedInMd` prop being set to true. 🤔

Comment (marbemac) [reply packages/elements-core/src/components/MarkdownViewer/CustomComponents/CodeComponent.tsx:78]: Oh my bad, nvm!

Comment (mpodlasin) [reply packages/elements-core/src/components/MarkdownViewer/CustomComponents/CodeComponent.tsx:78]: This code is part of `MarkdownViewer/CustomComponents` folder, where we store our overrides for markdown components.

**Review by wmhilton — APPROVED** [review]

**Review by marbemac — APPROVED** [review]

**Pull request merged**